### ISSUE TITLE
Duplicate Printers

### ISSUE THREAD

**Issue by Jcapehart2** [TFyre/bambu-farm#192]
Trying to deploy this to my farm but for some reason every printer shows up twice although the config only has them listed once.

I have 5 X1Cs, 10 A1, 1 A1 Mini and 16 P1S but they all show up twice. I also keep seeing connection refused for all the X1Cs. I know the video won't show up, that's fine. This is all running on a Pi. This is gonna make my farm so much easier to manage. Wish Panda Touch was desktop software instead!

**Comment by TFyre** [comment]
Please share your config after masking sensitive info 😄

**Comment by Jcapehart2** [comment]
`quarkus.http.host=0.0.0.0
quarkus.http.port=8080

bambu.users.admin.password=admin
bambu.users.admin.role=admin

# X1C Printers
bambu.printers.X1C-1.device-id=
bambu.printers.X1C-1.access-code=
bambu.printers.X1C-1.ip=

bambu.printers.X1C-2.device-id=
bambu.printers.X1C-2.access-code=
bambu.printers.X1C-2.ip=

bambu.printers.X1C-3.device-id=
bambu.printers.X1C-3.access-code=
bambu.printers.X1C-3.ip=

bambu.printers.X1C-4.device-id=
bambu.printers.X1C-4.access-code=
bambu.printers.X1C-4.ip=

# P1S Printers
bambu.printers.P1S-1.device-id=
bambu.printers.P1S-1.access-code=
bambu.printers.P1S-1.ip=

bambu.printers.P1S-2.device-id=
bambu.printers.P1S-2.access-code=
bambu.printers.P1S-2.ip=

bambu.printers.P1S-3.device-id=
bambu.printers.P1S-3.access-code=
bambu.printers.P1S-3.ip=

bambu.printers.P1S-4.device-id=
bambu.printers.P1S-4.access-code=
bambu.printers.P1S-4.ip=

bambu.printers.P1S-5.device-id=
bambu.printers.P1S-5.access-code=
bambu.printers.P1S-5.ip=

bambu.printers.P1S-6.device-id=
bambu.printers.P1S-6.access-code=
bambu.printers.P1S-6.ip=

bambu.printers.P1S-7.device-id=
bambu.printers.P1S-7.access-code=
bambu.printers.P1S-7.ip=

bambu.printers.P1S-8.device-id=
bambu.printers.P1S-8.access-code=
bambu.printers.P1S-8.ip=

bambu.printers.P1S-9.device-id=
bambu.printers.P1S-9.access-code=
bambu.printers.P1S-9.ip=

bambu.printers.P1S-10.device-id=
bambu.printers.P1S-10.access-code=
bambu.printers.P1S-10.ip=

bambu.printers.P1S-11.device-id=
bambu.printers.P1S-11.access-code=
bambu.printers.P1S-11.ip=

bambu.printers.P1S-12.device-id=
bambu.printers.P1S-12.access-code=
bambu.printers.P1S-12.ip=

bambu.printers.P1S-13.device-id=
bambu.printers.P1S-13.access-code=
bambu.printers.P1S-13.ip=

bambu.printers.P1S-14.device-id=
bambu.printers.P1S-14.access-code=
bambu.printers.P1S-14.ip=

bambu.printers.P1S-15.device-id=
bambu.printers.P1S-15.access-code=
bambu.printers.P1S-15.ip=

# A1 Printers
bambu.printers.A1-MINI.device-id=
bambu.printers.A1-MINI.access-code=
bambu.printers.A1-MINI.ip=

bambu.printers.A1-1.device-id=
bambu.printers.A1-1.access-code=
bambu.printers.A1-1.ip=

bambu.printers.A1-2.device-id=
bambu.printers.A1-2.access-code=
bambu.printers.A1-2.ip=

bambu.printers.A1-3.device-id=
bambu.printers.A1-3.access-code=
bambu.printers.A1-3.ip=

bambu.printers.A1-4.device-id=
bambu.printers.A1-4.access-code=
bambu.printers.A1-4.ip=

bambu.printers.A1-6.device-id=
bambu.printers.A1-6.access-code=
bambu.printers.A1-6.ip=

bambu.printers.A1-7.device-id=
bambu.printers.A1-7.access-code=
bambu.printers.A1-7.ip=

bambu.printers.A1-8.device-id=
bambu.printers.A1-8.access-code=
bambu.printers.A1-8.ip=

bambu.printers.A1-9.device-id=
bambu.printers.A1-9.access-code=
bambu.printers.A1-9.ip=

bambu.printers.A1-10.device-id=
bambu.printers.A1-10.access-code=
bambu.printers.A1-10.ip=
`

I am noticing it's showing the same printers twice the only difference is, one is a1-1 and the other is A1-1. or a1-mini and A1-MINI. Just upper/lower case versions but the env doesn't have two different versions 😅

**Comment by TFyre** [comment]
Thats probably happening because of upper/lowercase issues...

I would suggest rather using this format:
```
bambu.printers.printer1.name=A1-1
bambu.printers.printer1.device-id=
bambu.printers.printer1.access-code=
bambu.printers.printer1.ip=
```

https://github.com/TFyre/bambu-farm?tab=readme-ov-file#printer-section

**Comment by Jcapehart2** [comment]
Thank you, that did actually solve it. Guess I should've read better! Now to fight these X1Cs for video. Trying to deploy that solution it on a Pi so hopefully the steps aren't too specific for docker.

**Comment by TFyre** [comment]
Fixed in commit: 175ad98
